test/e2e/specs/fakerp/kubeletsecurity.go

```diff
@@ -0,0 +1,118 @@
+package fakerp
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	templatev1 "github.com/openshift/api/template/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+
+	"github.com/openshift/openshift-azure/pkg/util/random"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+	"github.com/openshift/openshift-azure/test/sanity"
+)
+
+const nginxTemplate = "nginx-example"
+
+func executeCommandInPod(client *openshift.Client, namespace, pod, container, command string) (string, string, error) {
+	var stdout, stderr bytes.Buffer
+	req := client.CoreV1.RESTClient().
+		Post().
+		Namespace(namespace).
+		Resource("pods").
+		Name(pod).
+		SubResource("exec").
+		VersionedParams(&corev1.PodExecOptions{
+			Command:   []string{"/bin/sh", "-c", command},
```

Comment:
are there any shell escaping libraries for `command`?  Just curious how the format of this string affects the outcome.